packages/server/database/migrations/20220121600507-addMoreRetroTemplates.ts

```diff
@@ -0,0 +1,667 @@
+    description: 'What did we learn?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Original 4', true),
+    question: 'Improvements ♻️ ',
+    description: 'What should we do differently next time?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Original 4', true),
+    question: 'Questions ❓',
+    description: 'What still puzzles us?',
+    sortOrder: 3
+  },
+  {
+    templateId: nameToId('Rose, Thorn, Bud 🌹', true),
+    question: 'Rose',
+    description: 'What went well? Which of our current practices or skills are strong?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Rose, Thorn, Bud 🌹', true),
+    question: 'Thorn',
+    description: 'What challenges or difficulties did we run into?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Rose, Thorn, Bud 🌹', true),
+    question: 'Bud',
+    description: 'What are our opportunities for growth and improvement?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Hopes and Fears 🎭', true),
+    question: 'Hopes 🙏',
+    promptId: 'hopesPrompt-2',
+    description: 'What do you hope for in the next period?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Hopes and Fears 🎭', true),
+    question: 'Fears 🧟 ',
+    description: 'What are you worried about for the next period?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Surprised, Worried, Inspired 😯', true),
+    question: 'Surprised',
+    description: 'What caught you off guard, positively or negatively?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Surprised, Worried, Inspired 😯', true),
+    question: 'Worried',
+    description: 'WWhat are you anxious about?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Surprised, Worried, Inspired 😯', true),
+    question: 'Inspired',
+    description: 'What has you energized for the future?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Superhero Retrospective 🦸', true),
+    question: 'Superpower',
+    description: 'What are your strongest skills?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Superhero Retrospective 🦸', true),
+    question: 'Gadgets',
+    description: 'What people, tools, or processes are most helpful?',
+    sortOrder: 1
+  },
+  {
+    templateId: nameToId('Superhero Retrospective 🦸', true),
+    question: 'Nemesis',
+    description: 'What makes your role more difficult?',
+    sortOrder: 2
+  },
+  {
+    templateId: nameToId('Superhero Retrospective 🦸', true),
+    question: 'Role',
+    description:
+      'Where do you fit into the team? How do you see your role interacting with others?',
+    sortOrder: 3
+  },
+  {
+    templateId: nameToId('Six Thinking Hats 🎩', true),
+    question: 'Blue Hat',
+    promptColor: PALETTE.SKY_400,
+    description: 'What do we want to achieve in this session?',
+    sortOrder: 0
+  },
+  {
+    templateId: nameToId('Six Thinking Hats 🎩', true),
+    question: 'White Hat',
+    promptColor: PALETTE.SLATE_100,
```

Comment:
I added the custom colours for Six Thinking Hats as requested by @garethaledavies